Android/samples/sdksample/app/src/main/java/com/microsoft/maps/sample/sdksample/MainActivity.java

```diff
@@ -211,6 +208,9 @@ private void addPin(Geopoint location, String title) {
         pushpin.setTitle(title);
         pushpin.setImage(mPinImage);
         pushpin.setNormalizedAnchorPoint(new PointF(0.5f, 1f));
+        if (title.isEmpty()) {
+            pushpin.setContentDescription("Pushpin");
```

Comment:
Perhaps add the lat, long coordinate? E.g.  "Pushpin at x y"

Reply:
Agreed offline to enumerate untitled pins instead.